src/libs/actions/Report.js

```diff
@@ -529,6 +608,8 @@ function fetchActions(reportID, offset) {
                 .max()
                 .value();
 
+            updateIOUReportData(reportID, indexedData);
```

Comment:
I don't really understand the first parameter here. Is it used? Why would it be passing a `reportID` only to call the API to get a different `reportID` to fetch with `API.GetIOUReport()`?

Reply:
Ah yes, this needs to be removed